.jscsrc

```diff
@@ -1,5 +1,5 @@
 {
-    "plugins": ["jscs-jsdoc"],
+    "plugins": ["../jscs-jsdoc"],
```

Comment:
We also can do something like `node node_modules/jscs/bin/jscs test lib`, but otherwise `npm run lint` is a way to do that.